packages/no-payment-integration/src/index.ts

```diff
@@ -1 +1,2 @@
+export { default as createNoPaymentStrategy } from './create-no-payment-strategy';
 export { default as NoPaymentDataRequiredPaymentStrategy } from './no-payment-data-required-strategy';
```

Comment:
Do we need to export this?

Reply:
Ignore this just had a look at core code, nice work on updating the references :D